rtest.py

```diff
@@ -213,7 +213,8 @@ def add_test(args):
     renaming them to the new test's name if necessary. Create a
     skeleton using the txt suite. Copy the skeleton to home/gold.
     Create a grammar using the current customization system, process
-    the profile. TODO: Add files to SVN.
+    the profile. It is the developer's responsibility to add these
```

Comment:
Yes! Much better than relying on some extra code to add files silently to version control (there was much confusion/issues previously, I think), and can use Sourcetree etc.

Reply:
Actually, right after I said it it turned out that TJ (I think) recently committed two incomplete tests (he forgot the gold). So, not doing this automatically is also problematic.

Reply:
> Actually, right after I said it it turned out that TJ (I think) recently committed two incomplete tests (he forgot the gold). So, not doing this automatically is also problematic.

I still don't think it's appropriate for the code to mess with its own version control mechanisms. Rather, I plan to write some actions that check if a push is missing the files. You can already see this with `python rtest.py --list --verbose` and look for the red "no"s, but I'll make it more easily discoverable.